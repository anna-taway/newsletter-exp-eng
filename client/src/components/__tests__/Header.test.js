import React from "react";
import Header from "../Header";
import { create } from "react-test-renderer";

describe("Header component", () => {
  test("Header component matches the snapshot", () => {
    const header = create(<Header />);
    expect(header.toJSON()).toMatchSnapshot();
  });
});