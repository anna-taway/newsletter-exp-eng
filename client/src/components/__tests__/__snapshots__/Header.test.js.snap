// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Header component matches the snapshot 1`] = `
<header
  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionRelative MuiAppBar-colorPrimary MuiPaper-elevation4"
>
  <div
    className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
  >
    <h6
      className="MuiTypography-root MuiTypography-h6 MuiTypography-colorInherit MuiTypography-noWrap"
    >
      #Newsletter
    </h6>
  </div>
</header>
`;
